pyvrp/solve.py

```diff
@@ -203,7 +203,7 @@ def solve(
 
     init = params.initial_solution
     if init is None:
-        init = ls(Solution(data, []), pm.max_cost_evaluator())
+        init = ls.search(Solution(data, []), pm.max_cost_evaluator())
```

Comment:
Use `search` because it ensures that all required clients are inserted in the initial pass.

Reply:
#963.